src/components/RangeSlider/components/SingleThumb/SingleThumb.scss

```diff
@@ -182,11 +178,6 @@ $range-output-spacing: rem(16px);
   transition-duration: duration();
   transition-timing-function: easing();
 
-  .Input:hover + &,
-  .Input:focus + & {
-    opacity: var(--p-override-zero);
-  }
```

Comment:
Same here. I could be wrong because I can't see all of the code so I'll make sure to 🎩 after review

Reply:
Totally! Anything come out of the 🎩?